doc/deploy.md

```diff
@@ -1,49 +1,48 @@
-# How to deploy a local environment
+# Deploy an xmtpd node to XMTP testnet
 
-**⚠️ Experimental:** warning, this file might be out of date!
+**⚠️ Experimental:** This file might be out of date!
 
-## XMTP Sepolia information
+The XMTP testnet environment lives in [XMTP Sepolia Chain](https://xmtp-testnet.explorer.alchemy.com/).
 
-The current testnet environment lives in [XMTP Sepolia Chain](https://xmtp-testnet.explorer.alchemy.com/).
+The XMTP testnet nodes operated by Ephemera include:
 
-### Deploy XMTPD nodes
-
-Node deployment is currently fully handled by [Ephemera](https://github.com/ephemeraHQ) and only members of `@ephemerahq/backend` have access to it.
-
-The nodes run by Ephemera are:
-
-| DNS Name                           | Location   | Public Key                                                           |
+| DNS name                           | Location   | Public key                                                           |
 | ---------------------------------- | ---------- | -------------------------------------------------------------------- |
 | https://grpc.testnet.xmtp.network  | US-EAST-2  | 0x03e5442c5d1fe2f02b6b9a1a386383a7766860b40a6079a0223994ffa2ce10512c |
 | https://grpc2.testnet.xmtp.network | EU-NORTH-1 | 0x02fc261d43a0153539a4c64c29763cb0e7e377c0eac2910c3d4bedb2235ac70371 |
 
-For more info, refer to the infrastructure README.
-
-## Local developer environment
-
-Refer to [XMTP Contracts](https://github.com/xmtp/smart-contracts) for further information regarding the on-chain protocol, deployments and source code.
+## Deploy a local development environment
 
-There are two ways of deploying a local environment:
+Use one of the following methods to deploy a local development environment:
```

Comment:
What is included in a local development environment? What gets deployed?

Reply:
Everything needed by xmtpd to run: databases, mls validation service, chain with all the contracts.